pandas/core/common.py

```diff
@@ -3095,3 +3095,32 @@ def pandas_dtype(dtype):
             pass
 
     return np.dtype(dtype)
+
+
+def validate_args(args, min_length=0, max_length=None):
+    """
```

Comment:
typo! I meant put a comment in ``core/common.py`` to indicate that this is a sub-section (for those 2 functions). ``core/common.py`` needs reorg, see #12503, and this will be an indication that these 2 functions belong together (you can also take suggestion from that issue and go ahead and put them in a new sub-module)